server/basicServices/basicServices/BasicServicesService.js

```diff
@@ -1407,7 +1407,7 @@ exports.updateOperationKey = function (body, user, originator, xCorrelator, trac
             isUpdated = await operationClientInterface.setOperationKeyAsync(operationUuid, newOperationKey);
           }
         } else {
-          throw new Error("OperationClientUuidisnotPresent")
+          reject(new BadRequestHttpException("OperationClientUuidisnotPresent"))
```

Comment:
Shall we make the reson for rejection 
from 
OperationClientUuidisnotPresent
to 
OperationClientUuid is not present

Reply:
done